docs/src/index.md

```diff
@@ -45,20 +45,20 @@ julia> problem = MISProblem(g)
 MISProblem(20)
 
 # select the branching strategy
-julia> branching_strategy = OptBranchingStrategy(TensorNetworkSolver(), IPSolver(), EnvFilter(), MinBoundarySelector(2), D3Measure())
-OptBranchingStrategy
+julia> branching_strategy = BranchingStrategy(TensorNetworkSolver(), IPSolver(), EnvFilter(), MinBoundarySelector(2), D3Measure())
```

Comment:
We should allow user to define other branching strategies, not only the optimal branching strategy.